### PR TITLE
Update links in readme and contributing files

### PR DESCRIPTION
**Pull request by D-system**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -5,16 +5,16 @@ New contributors are always welcome, when it doubt please ask questions. We stri
 ### Coding
 
 * Pick a task:
-  * Offer feedback on open [pull requests](https://github.com/heroku/platform/pulls).
-  * Review open [issues](https://github.com/heroku/platform/issues) for things to help on.
-  * [Create an issue](https://github.com/heroku/platform/issues/new) to start a discussion on additions or features.
+  * Offer feedback on open [pull requests](https://github.com/heroku/platform-api/pulls).
+  * Review open [issues](https://github.com/heroku/platform-api/issues) for things to help on.
+  * [Create an issue](https://github.com/heroku/platform/issues-api/new) to start a discussion on additions or features.
```
Comment (mikehale) [inline CONTRIBUTING.md]: typo here

File: README.md [hunk 2]
```diff
@@ -56,9 +56,9 @@ description of all available resources and methods.
 
 ### Handling errors
 
-The client uses [Excon](https://github.com/geemus/excon) under the hood and
+The client uses [Excon](https://github.com/excon/excon) under the hood and
 raises `Excon::Errors::Error` exceptions when errors occur.  You can catch specific
```
Comment (mikehale) [inline README.md]: replace `Excon::Errors::Error` with `Excon::Error`

File: CONTRIBUTING.md (hunk 1 above)
Comment (D-system) [inline CONTRIBUTING.md]: Nice catch, thank you

File: README.md (hunk 2 above)
Comment (D-system) [inline README.md]: Thank you for the heads up

**Comment by mikehale** [comment]
👍

**Pull request merged**